derp/derp_server.go

```diff
@@ -138,6 +138,10 @@ type Server struct {
 	// because it includes intra-region forwarded packets as the
 	// src.
 	sentTo map[key.Public]map[key.Public]int64 // src => dst => dst's latest sclient.connNum
+
+	// Stores empty content byte buffers after they've been used, so further allocations
+	// do not need to allocate more
+	bytes_pool *sync.Pool
```

Comment:
Go style doesn't use underscores: https://golang.org/doc/effective_go#mixed-caps

see also https://golang.org/s/style

Reply:
oops I've automatically got it ingrained it into me because of using python too much, my bad